google-cloud-bigtable-stats/src/main/java/com/google/cloud/bigtable/stats/StatsRecorderWrapper.java

```diff
@@ -59,9 +59,10 @@ public StatsRecorderWrapper(
     this.operationMeasureMap = statsRecorder.newMeasureMap();
   }
 
-  public void recordOperation(String status, String tableId, String zone, String cluster) {
+  public void recordOperation(
+      String status, String tableId, String zone, String cluster, String clientVersion) {
     TagContextBuilder tagCtx =
-        newTagContextBuilder(tableId, zone, cluster)
+        newTagContextBuilder(tableId, zone, cluster, clientVersion)
```

Comment:
In the BuiltinMetricsTracerFactory, you can add client version to statsAttributes so we don't need to update the APIs. I think this will simplify this PR a lot.